Source/dom/import.js

```diff
@@ -0,0 +1,35 @@
+import { wrapNativeObject } from './wrapNativeObject'
+
+function _importWithImporter(data, importer) {
+  importer.prepareToImportFromData(data)
```

Comment:
I'd rather consider `data` to be a Buffer (and also handle the case where it's an NSData) because `@skpm/fs` returns a Buffer (available through `require('buffer').Buffer`: https://github.com/skpm/buffer)